lib/pageflow/configuration.rb

```diff
@@ -95,6 +95,11 @@ class Configuration
     # @since 15.1
     attr_reader :entry_types
 
+    # Holds configurations for entry types.
+    # @return Hash
+    # @since 15.1
+    attr_accessor :entry_type_configs
+
```

Comment:
This can be a `attr_reader`. Let's also make it it `@api private` since it should not be used by outside code.